[WIP] issue-436: clarify-ssg-isg-statement (#450)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/introduction/about-guide.adoc

= About this guide

> The style and language guidance in this guide overrides or supplements some guidance provided by _IBM Style_, which is the primary source of style guidance for link:https://www.redhat.com/[Red Hat] product and cross-product solution documentation. 

Reviewer: To be consistent with how we reference it through the rest of the guide

```suggestion
The style and language guidance in this guide overrides or supplements some guidance provided by the _IBM Style_ guide, which is the primary source of style guidance for link:https://www.redhat.com/[Red Hat] product and cross-product solution documentation. 
```

Author: Fixed

> The style and language guidance provided in this guide overrides or supplements some guidance provided by _IBM Style_, which is the primary source of style guidance for link:https://www.redhat.com/[Red Hat] product and cross-product solution documentation. 

Reviewer 2: ```suggestion
The style and language guidance in this guide overrides or supplements some guidance provided by _IBM Style_, which is the primary source of style guidance for link:https://www.redhat.com/[Red Hat] product and cross-product solution documentation. 
```

Author: Changed per your suggestion

The Red{nbsp}Hat Customer Content Services team has created this guide as part of its responsibility for producing product documentation for Red{nbsp}Hat customers. Upstream communities who want to align more closely with the standards used by Red{nbsp}Hat product documentation can also use this guide.

[NOTE]
====
Other Red Hat technical documentation, including Red Hat training and exam content by Global Learning Services (GLS), follows the link:https://stylepedia.net/[_Red Hat Technical Writing Style Guide_] instead of the _Red{nbsp}Hat supplementary style guide for product documentation_.
====

== Using style guides for Red{nbsp}Hat product documentation

> Red{nbsp}Hat product documentation adheres to the style guidance provided in _IBM Style_, the _Red{nbsp}Hat supplementary style guide for product documentation_, and style guides specific to Red{nbsp}Hat products, such as Red{nbsp}Hat Enterprise Linux or Red{nbsp}Hat Openshift Container Platform.

Reviewer: Same about the ISG wording. And also I think just one example of a product-specific guide is sufficient. Esp since we don't talk about the RHEL one again, but we do the OCP one in the graphic.

Also, do you think we should link to the OCP one since it's public? I don't feel strongly about it so I didn't include it in my suggestion below, just just something to consider.

```suggestion
Red{nbsp}Hat product documentation adheres to the style guidance provided in the _IBM Style_ guide, the _Red{nbsp}Hat supplementary style guide for product documentation_, and style guides specific to Red{nbsp}Hat products, such as the Red{nbsp}Hat Openshift documentation guidelines.
```

Author: I included those two because those are the two biggest sets of RH product docs. I'd prefer to leave a link out because we don't provide links for anything else in this section. So leaving it as-is.

> Red{nbsp}Hat product documentation adheres to the style guidance provided in _IBM Style_, the _Red{nbsp}Hat supplementary style guide for product documentation_, and style guides specific to Red{nbsp}Hat products, such as Red{nbsp}Hat Enterprise Linux or Red{nbsp}Hat Openshift Container Platform.

Reviewer 3: ```suggestion
Red{nbsp}Hat product documentation adheres to the style guidance provided in the _IBM Style_ guide, the _Red{nbsp}Hat supplementary style guide for product documentation_, and style guides specific to Red{nbsp}Hat products, such as the Red{nbsp}Hat OpenShift documentation guidelines.
```
Fixed caps in Open**S**hift

Author: fixed

> When seeking style guidance, consult the style guide specific to your product documentation project first, if applicable, then the _Red{nbsp}Hat supplementary style guide for product documentation_, and finally _IBM Style_.

Reviewer: ```suggestion
When seeking style guidance, consult the style guide specific to your product documentation project first, if applicable, then the _Red{nbsp}Hat supplementary style guide for product documentation_, and finally the _IBM Style_ guide.
```

Author: Fixed

> When seeking style guidance, consult the style guide specific to your product documentation project first, if applicable, and then the _Red{nbsp}Hat supplementary style guide for product documentation_, and then _IBM Style_.

Reviewer 2: ```suggestion
When seeking style guidance, consult the style guide specific to your product documentation project first, if applicable, then the _Red{nbsp}Hat supplementary style guide for product documentation_, and finally _IBM Style_.
```

Author: Edited per your suggestion. Thanks!

> The following image illustrates the relationship between the different style guides in Red{nbsp}Hat product documentation:

Reviewer: Instead of just "relationship", would it be more accurate to say that it demonstrates the order?

```suggestion
The following image illustrates the order to consult the different style guides in Red{nbsp}Hat product documentation:
```

Author: Changed to "hierarchical relationship" because it's not just about the order to use when consulting them.

image:images/ccs-style-guide-hierarchy.png[CCS style guide hierarchy]

> If you think that a documentation use case needs to deviate from the guidance in either guide, notify the style council by opening an link:https://github.com/redhat-documentation/doc-style/issues[issue]. This way, the deviation can be discussed by the style council and included in this guide.

Author: Changed
